src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -50,6 +53,7 @@ void *frontendservicesesion(std::string masterIP, int connFd, SQLiteDBInterface
     frontend_logger.log("Master IP: " + masterIP, "info");
     char data[FRONTEND_DATA_LENGTH];
     bzero(data, FRONTEND_DATA_LENGTH + 1);
+    DataPublisher workerClient(7780, "127.0.0.1");
```

Comment:
Why is this host IP and port hard coded here?

Reply:
Fixed in https://github.com/miyurud/jasminegraph/pull/122/commits/7174d9c7814f01e19de59293f8917ecd669a662f

Reply:
Ok, done